src/styles/LoginPage.scss

```diff
@@ -1,331 +1,351 @@
+      }
     }
-  }
+    .login__form__input {
+      .first__input {
+        display: flex;
+        flex-direction: column;
 
-  .login__form__desc {
-  }
-  .login__form__button {
-    display: flex;
-    margin-bottom: 2rem;
-    .login__google {
-      button {
-        box-sizing: border-box;
-        width: 6rem;
-        height: 50px;
-        // padding: 0.3rem 2rem 0rem 2rem;
-        border: 2px solid #717ff8;
-        border-radius: 10px;
-        background: #ffffff;
-        position: relative;
-        cursor: pointer;
-        text-indent: 100%;
-        white-space: nowrap;
-        overflow: hidden;
-        color: #ffffff;
-        img {
+        input {
+          width: 40rem;
+          height: 4rem;
           box-sizing: border-box;
-          width: 30px;
-          height: 30.7px;
-          position: absolute;
-          left: 2rem;
-          bottom: 0.5rem;
+          padding: 0rem 0rem 0rem 1.2rem;
+          border-radius: 0.8rem;
+          border: 2px solid #ffffff;
+          box-shadow: 1px 1px 3px 2px rgba(0, 0, 0, 0.25);
+          margin: 2rem 0rem 0rem 0rem;
+          font-size: 1.2rem;
+          font-weight: 400;
         }
-        .login__btn__text {
-          margin-left: 1rem;
+      }
+      .second__input {
+        display: flex;
+        flex-direction: column;
+        box-sizing: border-box;
+        input {
+          width: 40rem;
+          height: 4rem;
+          box-sizing: border-box;
+          padding: 0rem 0rem 0rem 1.2rem;
+          border-radius: 0.8rem;
+          border: 2px solid #ffffff;
+          box-shadow: 1px 1px 3px 2px rgba(0, 0, 0, 0.25);
+          margin: 2rem 0rem 0rem 0rem;
+          font-size: 1.2rem;
           font-weight: 400;
-          font-size: 15px;
         }
-        &:hover {
-          background-color: #717ff8;
-          color: #ffffff;
+        a {
+          font-size: 1.2rem;
+          text-decoration: none;
+          margin-left: 67%;
+          margin-top: 0.5rem;
         }
       }
     }
-    .login__apple {
-      margin-left: 3rem;
+    .login__button {
+      width: 40rem;
+      height: 4rem;
+      border-radius: 0.8rem;
+      background-color: #717ff8;
+      border: none;
+      font-size: 1.3rem;
+      color: #ffffff;
+      margin: 0rem 0rem 2rem 0rem;
+      &:hover {
+        background-color: #30a2ff;
+      }
+    }
+  }
+}
 
-      button {
-        box-sizing: border-box;
-        width: 6rem;
-        height: 50px;
-        padding: 0.3rem 0rem 0rem 2rem;
-        border: 2px solid #717ff8;
-        border-radius: 10px;
-        background: #ffffff;
-        position: relative;
-        cursor: pointer;
-        text-indent: 100%;
-        white-space: nowrap;
-        overflow: hidden;
-        img {
-          width: 29px;
-          height: 33px;
-          margin-right: 1rem;
-          position: absolute;
-          left: 2rem;
-          bottom: 0.5rem;
+//
+//
+//
+//Mobile or tablet view
+[class^="login__form"] {
```

Comment:
This line code make the all the code in tablet and mobile not read the readiness. Change it to the media breakpoint to read the code.